assets/js/googlesitekit/notifications/datastore/notifications.js

```diff
@@ -359,6 +437,30 @@ export const reducer = createReducer( ( state, { type, payload } ) => {
 			break;
 		}
 
+		case MARK_NOTIFICATION_SEEN: {
+			const { notificationID, currentDate } = payload;
```

Comment:
`currentDate` is a bit of a strange name for this parameter; if the value should _always_ be the current date then there's no sense in having it be a param.

But if it can be any date, then `date` or `dateSeen` is a much clearer parameter name. I agree that it should accept an argument, but let's update it to be a more clear name.

```suggestion
			const { dateSeen, notificationID } = payload;
```